sys/evtimer/evtimer.c

```diff
@@ -210,10 +199,11 @@ void evtimer_init(evtimer_t *evtimer, evtimer_callback_t handler)
 void evtimer_print(const evtimer_t *evtimer)
 {
     evtimer_event_t *list = evtimer->events;
+    int nr = 0;
 
-    while (list->next) {
-        evtimer_event_t *list_entry = list->next;
-        printf("ev offset=%u\n", (unsigned)list_entry->offset);
+    while (list) {
```

Comment:
Am I wrong, or did the old code fail to print the first entry? Anyway, this code does what it should.

Reply:
Yes, and that is why I have a second commit for that